beprof/curve.py

```diff
@@ -43,6 +45,7 @@ class Curve(np.ndarray):
     """
 
     def __new__(cls, input_array, **meta):
+        logging.info('Creating Curve object, metadata is: %s', meta)
```

Comment:
You are using Python2 string formatting. In Python3 it has been replaced with new one. Why was it done is described here: https://www.python.org/dev/peps/pep-3101/ . How to use it, read this: https://docs.python.org/3.5/library/string.html#formatstrings